### ISSUE TITLE
Support protobuf version `3.20.0`

### ISSUE THREAD

**Issue by mharrisb1** [googleapis/python-run#70]
**Is your feature request related to a problem? Please describe.**
Currently can't install with `apache-airflow-providers-google` due to incompatible supported versions of protobuf. 

**Describe the solution you'd like**
Add support for version `3.20.0` to match requirements (https://pypi.org/project/apache-airflow-providers-google/)

**Comment by parthea** [comment]
Closing as obsolete. Protobuf >=3.19.5 is supported.
https://github.com/googleapis/python-run/blob/main/setup.py#L41

**Comment by mharrisb1** [comment]
@parthea it looks like it is explicitly excluding v3.20.0?

**Comment by parthea** [comment]
You’re right. We are explicitly excluding 3.20.0 and 3.20.1 due to the following bulletin:
https://cloud.google.com/support/bulletins#GCP-2022-019

Please use 3.19.5 or 3.20.2 as recommended in the bulletin.

**Comment by berkcoker** [comment]
This is still preventing us from using this library in the Composer environment. Has anyone figured out how to make it work?

**Comment by parthea** [comment]
The latest version already supports protobuf 3.19.5 and 3.20.2
https://github.com/googleapis/python-run/blob/2bb539896cb989831edb36acb2d4b1beb69b18a5/setup.py#L42

protobuf 3.20.0 and 3.20.1 will not be supported in new releases because of the security bulletin
https://cloud.google.com/support/bulletins#GCP-2022-019

**Comment by berkcoker** [comment]
Those two versions cannot be used in the Composer environment, as google-ads doesn't allow for 3.19.* and >3.20.0

**Comment by parthea** [comment]
Googlers see http://b/282956119

**Comment by crisol3036** [comment]
> Googlers see b/282956119

@parthea I'm unfamiliar with that notation, I thought it could mean `b` for `branch` but checked the branches and didn't find that number anywhere. What does it mean?

**Comment by algarci** [comment]
We are facing the same issue here. Any update??
